quickwit/quickwit-metastore/src/metastore/mod.rs

```diff
@@ -855,6 +860,12 @@ impl ListSplitsQuery {
         self.sort_by = SortBy::IndexUid;
         self
     }
+
+    /// Only return splits whose (index_uid, split_id) are lexicographically after this split
+    pub fn after_split(mut self, split_meta: &SplitMetadata) -> Self {
```

Comment:
can we a have a unit test in the metastore suite that checks:
- that after_split works as expected.
- that the results are sorted by index uid AND split_id